README.md

```diff
@@ -24,3 +24,9 @@ Import-Module -Name '.\Carbon.AccountManagement'
 ```
 
 # Commands
+
+* `Resolve-Identity`: Determines the full, NT identity name for a user or group.
+* `Resolve-IdentityName`: Resolves a user/group name into its full, canonical name, used by the operating system.
+* `ConvertTo-SecurityIdentifier`: Converts a string or byte array security identifier into a 
+System.Security.Principal.SecurityIdentifier Object.
+* `Test-Identity`: Tests that a name is valid Windows local or domain user/group.
```

Comment:
Document commands with the default module prefix, `C`. For example: `Test-CIdentity`